frontend/src/app/footer/footer.component.css

```diff
@@ -0,0 +1,11 @@
+ 
+  .footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    /* Set the fixed height of the footer here */
+    height: 60px;
+    line-height: 60px;
+    /* Vertically center the text there */
+    background-color: #f5f5f5;
+  }
```

Comment:
Footer is not included in any file and there for should not included into this PR

Reply:
Having the files in there makes it easier to include it in the future